docs/operator-docs/cumulus-dependencies.md

```diff
@@ -0,0 +1,72 @@
+---
+id: cumulus-dependencies
+title: Cumulus Dependencies
+hide_title: true
+---
+
+# Updating Cumulus Dependencies
+
+
+## Cumulus Dependency Management - A Quick Intro to package.json
+
+As is standard with nodejs projects, Cumulus has lists of dependencies in `package.json` files. For this example, we'll look at the `package.json` file under the `example` directory in the main [Cumulus project](https://github.com/nasa/cumulus).
+
+In the `package.json` file, both the `dev_dependencies` and `dependencies` attributes should exist. The `dev_dependencies` are packages that are used in development, but are not necessary for deployment/usage of the package; a good example of this would be testing frameworks (`ava`) or mocking packages. The `dependencies` attribute lists all the required packages for using the software. Cumulus core developed dependencies all are prefixed with `@cumulus/` (i.e. the ingest package is listed as `@cumulus/ingest`).
+
+Example:
+```json
+...
+dependencies: {
+  "@cumulus/api": ^1.11.1,
+  "@cumulus/ingest": ^1.11.1,
+  ...
+}
+```
+
+A detailed explanation of the dependency version syntax (`^1.11.1`) can be found in the npm package.json [documentation](https://docs.npmjs.com/files/package.json#dependencies). That page also has detailed documentation outlining many facets of the `package.json`.
```

Comment:
I think we should spell out what the carat syntax does here, since it's the default.  I'm also questioning a bit whether we want to encourage people to use it, rather than specifying a particular release.  I think we do, but we need to be really explicit what it means.

Reply:
On line 53 I explain the carat, but I could see it being valuable bringing that explanation up a bit.